custom_components/powercalc/data/lenovo/Smart Clock with Google Assistant/model.json

```diff
@@ -0,0 +1,25 @@
+{
+	"measure_description": "Manually sampled at 10% volume increases, radio, listening to EDM. Max power is from manufacturer specs",
+	"measure_method": "manual",
+	"measure_device": "TP-Link Tapo P110",
+	"name": "Lenovo Smart Clock with Google Assistant",
+	"standby_power": 2.511,
+	"requires_additional_configuration": true,
+	"device_type": "media_player",
+	"supported_modes": ["linear"],
+	"linear_config": {
+		"calibrate": [
+			"10 -> 3.12",
+			"20 -> 3.28",
+			"30 -> 3.34",
+			"40 -> 3.35",
+			"50 -> 3.47",
+			"60 -> 3.46",
+			"70 -> 3.48",
+			"80 -> 4.28",
+			"90 -> 4.33",
+			"100 -> 4.41"
+		],
+		"max_watt": 6.00
```

Comment:
This won't be used when the calibrate mode is used. It's one or the other.
So it can be removed.